packages/api/test/dist.test.ts

```diff
@@ -26,4 +27,22 @@ describe('typescript dist verification', function () {
 
     mock.done();
   });
+
+  it('should be able to set an auth token', async function () {
```

Comment:
Added this new test because the `.auth()`, `.config()` and `.server()` accessors don't talk to the specs cache so I needed to make sure that the new APICore class was being handled properly there in the dist.